etc/proto/pachgen/transaction_client.go

```diff
@@ -0,0 +1,108 @@
+//nolint:wrapcheck
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"path"
+	"strings"
+	"text/template"
+
+	descriptor "github.com/golang/protobuf/protoc-gen-go/descriptor"
+	plugin "github.com/golang/protobuf/protoc-gen-go/plugin"
+)
+
+func init() {
+	CodeGenerators = append(CodeGenerators, &TransactionClientGenerator{})
+}
+
+type TransactionClientGenerator struct {
+	Protos []*descriptor.FileDescriptorProto
+}
+
+var funcs = map[string]interface{}{
+	"importPath": func(proto *descriptor.FileDescriptorProto) string {
+		if *proto.Package == "taskapi" {
```

Comment:
The fact that the proto package is called `taskapi` while the Go package in `src/` is called `task` makes this import a bit awkward.

Reply:
the way protoc handles this is to name every single import `<protoPackageName> <go/package/path>`, so you don't have to special case. I'd prefer that here

Reply:
For proto packages with `_v2`, should we remove the `_v2` or import as-is?